build_process/package-management.md

```diff
@@ -59,19 +59,19 @@ To quickly create a new package run `npm init -y --scope sparkbox`. This will cr
 }
 ```
 
-Private packages will typically be used in client work (design systems, component libraries, various utilities). Consider scoping these to something company-appropriate to avoid naming collisions with open-source packages.
+We typically use private packages in client work (design systems, component libraries, various utilities). Consider scoping these to something company-appropriate to avoid naming collisions with open-source packages.
 
-Additional fields that we will most likely want to use in package.json:
-- [module][pkgModule] - offer an ES module entry point
+These are additional fields that we usually want to use in package.json:
+- [module][pkgModule] — offer an ES module entry point
 - [files][pkgFiles] - don't package more than we need
 - [engines][pkgEngines] - tell people and machines what node/npm version we expect
 
 Read more about [package.json][pkg] fields.
```

Comment:
Could you describe a little more of what they'll find if they follow this link to learn more? Seems like a short description could be helpful to readers.